packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -6907,84 +6917,78 @@ export default class Meeting extends StatelessWebexPlugin {
   }
 
   /**
-   * Publishes a track.
+   * Publishes a stream.
    *
-   * @param {LocalTrack} track to publish
+   * @param {LocalStream} stream to publish // TODO: this should probably be required...
+   * @param {MediaType} mediaType of the stream // TODO: ...and this should be required too
    * @returns {Promise}
    */
-  private async publishTrack(track?: LocalTrack) {
-    if (!track) {
+  private async publishStream(stream?: LocalStream, mediaType?: MediaType) {
+    if (!stream) {
       return;
     }
 
-    if (this.mediaProperties.webrtcMediaConnection) {
-      if (this.isMultistream) {
-        await this.mediaProperties.webrtcMediaConnection.publishTrack(track);
-      } else {
-        track.setPublished(true); // for multistream, this call is done by WCME
-      }
+    if (this.isMultistream && this.mediaProperties.webrtcMediaConnection) {
+      await this.sendSlots.get(mediaType).publishStream(stream);
     }
   }
 
   /**
-   * Un-publishes a track.
+   * Un-publishes a stream.
    *
-   * @param {LocalTrack} track to unpublish
+   * @param {LocalStream} stream to unpublish // TODO: this should probably be removed...
```

Comment:
When we say probably, do we have some unknowns here?

Reply:
Ah, I remember writing this comment. Not sure if the check for whether or not `stream` exists is necessary, but if it's not then we should remove this.